index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en" data-theme="light">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Document</title>
+		<link href="https://cdn.jsdelivr.net/npm/daisyui@3.5.1/dist/full.css" rel="stylesheet" type="text/css" />
+		<script src="https://cdn.tailwindcss.com"></script>
+	</head>
+	<body class="w-screen min-h-screen md:flex bg-slate-100">
+		<div class="flex card card-side w-full md:m-10 m-5 shadow-lg overflow-hidden p-5 bg-white">
+			<div class="w-1/4 hidden bg-purple-800 md:flex card p-5"></div>
+			<form class="h-full w-full md:w-3/4 py-7 justify-between flex flex-col md:mx-12 gap-10 md:gap-0">
```

Comment:
Twoje klasy odnoszą się bezpośrednio do tego co masz w stylach - to nie jest dobry pomysł. Co jeśli uznasz po X latach, że odległość między elementami ma się zwiększyć o 5px? Będziesz zmieniał w HTML'u tagże? Szkoda rąk :).